### PR TITLE
Add Post about The Power of Believing Other People Believe

### PR DESCRIPTION
**Pull request by alexandre-normand**
@coderunner: I thought of trying something new and having a pre-publish review of a potential future blog post. I know the blog is all mine and I should be able to do my own editing but I thought this could be a fun experiment. 

At a high-level, I'm trying to get momentum being writing more regularly and, while I'm not totally convinced this one is worth posting, I think it might be useful in getting the ball rolling. However, it's also possible that I could feel good about this one if I found the key element I'm missing. 

Or maybe I should just start writing in French and try that as an exercise?

### CODE REVIEW

File: content/the-power-of-believing-other-people-believe.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Could we consider a do-over or a gradual simplification? Could we get back to a time without the money abstraction? What’s interesting about those questions is that the end goal is realively easy to imagine. After all, we roughly know what it would be like to live without money. For some examples, you could refer to stories of natural catastrophes such as the ones from [A Paradise Built in Hell](https://www.amazon.com/Paradise-Built-Hell-Extraordinary-Communities/dp/0143118072/ref=sr_1_1?ie=UTF8&qid=1544593296&sr=8-1&keywords=A+Paradise+Built+in+Hell). Essentially, we’d focus on collaborating with our immediate neighbors to meet our most essential needs without the need for money exchange. What is more difficult to imagine is the process of it happening organically of our own free will. 
+
+To be fair, the [gift economy](https://en.wikipedia.org/wiki/Gift_economy) is an approximation of this to some degree. But without more radical change where property and/or taxes cease to exist, money remains. 
+
+{{% interlude %}}
```
Comment (alexandre-normand) [inline content/the-power-of-believing-other-people-believe.md]: That one is a custom `hugo` `shortcode` I wrote to render the paragraph in a way similar to the beginning on [this post](https://heyitsalex.net/throwing-away-some-ideas-about-recycling/).

File: content/the-power-of-believing-other-people-believe.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+But everything is a tradeoff. By creating this abstraction, we promoted the accumulation of wealth[ˆ1]. By creating this abstraction, we lost a direct connection to our most important and immediate needs (because as long as you have money, you’ll be able to get by). By creating this abstraction, we lost sight of the real value of what we were trading and giving up: our time, our health, the pleasure to struggle and be uncomfortable and, in other words, our selves. 
+
+I’ve said this in previous posts but I believe it’s time to reconsider those abstractions. In software engineering, it’s a common pattern for complexity to creep in via abstractions built on top of abstractions. Well designed ones enable the cycles to go on withsome success. Until it doesn’t. It’s a common story for teams to feel the pain of maintaining such complexity and go on quests for simplication. Because of the smaller scope of software, it’s usually easier to imagine and realize. It can also be justified by rational goals. Let’s build something simpler that we can easily debug and troubleshoot what should now match our current requirements better.
```
Comment (coderunner) [inline content/the-power-of-believing-other-people-believe.md]: `withsome`

**Comment by coderunner** [comment]
👍
I ❤️ it!

**Comment by alexandre-normand** [comment]
Thanks for the review, @coderunner!

**Pull request merged**